doc/src/vpr/file_formats.rst

```diff
@@ -391,17 +391,34 @@ The ``.param`` statement allows parameters (e.g. primitive modes) to be tagged o
 
 .. note:: ``.param`` statements apply to the previous primitive instantiation.
 
+Parameters can have one of the three available types. Type is inferred from the format in which a parameter is provided.
+
+ * **string**
+    Whenever a parameter value is quoted it is considered to be a string.
+
+ * **binary word**
+    Binary words are specified using strings of characters ``0`` and ``1``. No other characters are allowed. Number of characters denotes the word length.
+
+ * **real number**
```

Comment:
What is `.param x 2`?  A real number or error?

Reply:
Its an error. For an integer 2 you need `10`, for a real 2 you need `2.0`.

Reply:
I think this needs to make clear that `2` is illegal, because it is not a binary word (has a 2) and is not a string (no quotes) and not a real number (no `.`).

Reply:
Added clearer explanation of illegal formats